cw_nano/lib/banano_balance.dart

```diff
@@ -1,11 +1,27 @@
 import 'package:cw_core/balance.dart';
 import 'package:nanoutil/nanoutil.dart';
 
-class BananoBalance extends Balance {
+BigInt stringAmountToBigIntBanano(String amount) {
+  return BigInt.parse(NanoAmounts.getAmountAsRaw(amount, NanoAmounts.rawPerBanano));
+}
+
+class NanoBalance extends Balance {
   final BigInt currentBalance;
   final BigInt receivableBalance;
 
-  BananoBalance({required this.currentBalance, required this.receivableBalance}) : super(0, 0);
+  NanoBalance({required this.currentBalance, required this.receivableBalance}) : super(0, 0);
```

Comment:
should stay Banano I think? or might even find a better way since the only change is the function above